### PR TITLE
PhpdocIndentFixer - Fix for open tag

### PR DESCRIPTION
**Pull request by SpacePossum**
This PR addresses this issue:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/1407

### CODE REVIEW

File: Symfony/CS/Fixer/Symfony/PhpdocIndentFixer.php [hunk 1]
```diff
@@ -40,7 +40,8 @@ public function fix(\SplFileInfo $file, $content)
 
             // ignore inline docblocks
             if (
-                ($prevToken->isWhitespace(array('whitespaces' => " \t")) && !$tokens[$index - 2]->isGivenKind(T_OPEN_TAG))
+                $prevToken->isGivenKind(T_OPEN_TAG)
+                || ($prevToken->isWhitespace(array('whitespaces' => " \t")) && !$tokens[$index - 2]->isGivenKind(T_OPEN_TAG))
```
Comment (GrahamCampbell) [inline Symfony/CS/Fixer/Symfony/PhpdocIndentFixer.php]: surely we should allow other kinds of whitespace too?

Comment (GrahamCampbell) [inline Symfony/CS/Fixer/Symfony/PhpdocIndentFixer.php]: oh, wait, ignore me, lol

**Comment by keradus** [comment]
looking nice ;)

**Comment by GrahamCampbell** [comment]
This if statement is somewhat hard to read. Maybe it's worth moving it out to a protected method with multiple logical blocks in it rather than one massive ass statement.

**Comment by SpacePossum** [comment]
@GrahamCampbell I don't see much value adding the overhead with a method. What do you think @keradus ?

**Comment by keradus** [comment]
Each condition is written in separate line, if is clear for me.

But I'm also a perl developer, where you used to write code like
```perl
@dst = sort { $$a{nested_key} <=> $$b{nested_key} } map { $$_{flat_key} } @src if ($$one{need}[$the]{data});
````
;)

**Comment by GrahamCampbell** [comment]
Well, I managed to missread it the first time, so I didn't find it very clear.

**Comment by GrahamCampbell** [comment]
It made the diff hard to read in particular, more than the code before and after.

**Comment by SpacePossum** [comment]
I see no problem with the Perl code ^-^

The diff. might be a bit hard, but as long as the end code is OK I think I rather keep this PR as is.

**Comment by GrahamCampbell** [comment]
But, the point in a lot of symfony's code style, IS to make diffs easier to read, and I think that's very important.

**Comment by SpacePossum** [comment]
I agree it is important, however I think most developers spend more time reading the code then the diffs of it.
It is not an excuse to mess up the history, but in this case I think the readability of the code outweighs that of the diff.

**Comment by GrahamCampbell** [comment]
I think it would make BOTH more readable.

**Comment by keradus** [comment]
I think that discussion has little point now...

**Comment by stof** [comment]
> But, the point in a lot of symfony's code style, IS to make diffs easier to read, and I think that's very important.

Trailing commas in multiline arrays and not aligning assignment/array arrows are not about making the diff easier to read. They are about avoiding useless changes in diffs, which would also lead to more frequent conflicts (even more important in Symfony as we merge older branch into newer branches all the time, and we try to avoid conflicts between maintained branches whenever possible. this is also why we don't reorder existing use statements)

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Comment by keradus** [comment]
Merged into 1.10, not 1.11 as requested.

**Comment by SpacePossum** [comment]
thanks @keradus and the reviewers :)
sorry for targeting the wrong branch : |

**Pull request closed without merging**